deploy/crds/kustomize_operatorpolicy/policy.open-cluster-management.io_operatorpolicies.yaml

```diff
@@ -261,6 +261,8 @@ spec:
                           description: Whether the object was created by the parent
                             policy
                           type: boolean
+                        diff:
+                          type: string
```

Comment:
Can we remove this from OperatorPolicy? I think it should be easy with another patch in its kustomization.